example/main.tf

```diff
@@ -1,3 +1,8 @@
+provider "vault" {
+  address = "http://127.0.0.1:8200"
+  token = "master"
+}
+
```

Comment:
Better move this block under providers.tf .
Also, could you set up the providers version explicitly